### PR TITLE
Modify deltablue to avoid gratuitous exception at runtime

### PR DESCRIPTION
**Pull request by AndyAyersMS**

### CODE REVIEW

**Comment by AndyAyersMS** [comment]
Related to dotnet\llilc#1000

@JosephTremoulet PTAL

File: tests/src/JIT/Performance/CodeQuality/V8/DeltaBlue/DeltaBlue.cs [hunk 1]
```diff
@@ -906,17 +906,22 @@ public static void Bench()
...
         int iterations = 200; // read iterations from arguments, walter 7/97
-        try
-        {   // read iterations from arguments, walter 7/97
-            iterations = Int32.Parse(args[0]);
-        }
-        catch (Exception)
+        if (args.Length > 0)
         {
+            bool parsed = Int32.TryParse(args[0], out iterations);
+            if (! parsed)
```
Comment (JosephTremoulet) [inline tests/src/JIT/Performance/CodeQuality/V8/DeltaBlue/DeltaBlue.cs]: `! parsed` looks a little funny (as opposed to `!parsed`), don't know if that was intentional.

**Comment by JosephTremoulet** [comment]
LGTM, one style nit

**Comment by AndyAyersMS** [comment]
codeformatter agrees with you -- updated in place.

**Pull request merged**